dashboard/test/models/course_offering_test.rb

```diff
@@ -438,7 +438,7 @@ class CourseOfferingTest < ActiveSupport::TestCase
   end
 
   test "can serialize and seed course offerings" do
-    course_offering = create :course_offering, key: 'course-offering-1'
+    course_offering = create :course_offering, key: 'course-offering-1', grade_levels: 'K,1,2', curriculum_type: 'Course', marketing_initiative: 'Hour of Test', header: 'Test'
```

Comment:
Do we want to build in any sort of backend protection so that the inputs can only be those from the shared constants?

Reply:
Perfect- thank you!

Reply:
For this test then, do you want to give inputs that would automatically pass/be acceptable inputs?

Reply:
oh good call! I'll update!